src/login.js

```diff
@@ -18,24 +18,22 @@ const inquirer = require('inquirer');
 const configure = require('./configure.js');
 const saml = require('./utils/SamlUtil.js');
 const AwsCliUtil = require('./utils/AwsCliUtil.js');
-const os = require('os');
-const path = require('path');
+const KEYHUB_CONFIG_DIR = require('./configure.js').getKeyhubConfigDir();
 let samlPayloadIntercepted = false;
 
 module.exports = {
 
-    login: async function () {
+    login: async function (options) {
         try {
             await configure.hasValidConfiguration();
+            await AwsCliUtil.checkIfConfigFileExists();
         } catch (error) {
-            console.error('No valid configuration found.');
             console.error(error.message);
-            console.error('Please run `aws-keyhub -c` first.');
             process.exit(1);
         }
 
         const browser = await puppeteer.launch({
-            userDataDir: os.homedir() + path.sep + '.aws-keyhub' + path.sep + 'puppeteer_profile'
+            userDataDir: KEYHUB_CONFIG_DIR + 'puppeteer_profile/'
```

Comment:
I would say we move als the paths to one central location for example configure.js or a new Util class. And also you are using / instead of path.sep here.